platform/i18n/src/locales/en-US/index.js

```diff
@@ -1,25 +1,23 @@
-import AboutModal from './AboutModal.json';
 import Buttons from './Buttons.json';
 import CineDialog from './CineDialog.json';
 import Common from './Common.json';
-import DatePicker from './DatePicker.json';
 import Header from './Header.json';
 import MeasurementTable from './MeasurementTable.json';
+import SidePanel from './SidePanel.json';
+import StudyBrowser from './StudyBrowser.json';
```

Comment:
why we are removing AboutModal and DatePicker etc. from this file?

Reply:
Excuse Me, it's true, this is an error. I have pushed the fix 889a71f

Reply:
Thanks, also the viewportDownloadForm

Reply:
Done :)